index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+   <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+   <header>
+      <nav>
+         <div class="logo"><a href="#">I<span>.J</span></a></div>
+         <div class="hamburger-menu"><i class="fa-solid fa-bars"></i></div>
+      </nav>
+
+        <section class="title">
+          <h1>I'm Isaac. Glad To see you!</h1>
+        
+          <p>
+            I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
+            experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+           
+          <h2>LET'S CONNECT</h2>
+         
+          <ul class="socials">
+            <li>
+              <a href="https://twitter.com/ejama_isaac">
+               <img src="img/Vector (3).svg" alt="">
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/ejama-isaac-bab607267/"><img src="img/Linkedin icon.svg" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="img/Vector (2).svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="img/Vector.svg" alt=""></a>
+            </li>
+            <li>
+              <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
```

Comment:
- [ ] As noted by the last reviewer, kindly work on the spacing between the icons. This is ideal for a better user experience from the side of the end user and makes your code looks great 👍 

## Your design:
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/50990925/a3ab51fe-4766-41bb-9b65-168f9782e048)

## Figma:
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/50990925/a2dd87e9-51d5-4d44-8e2d-3784691bd696)